packages/block-library/src/utils/caption.js

```diff
@@ -32,6 +32,7 @@ export function Caption( {
 	placeholder = __( 'Add caption' ),
 	label = __( 'Caption text' ),
 	showToolbarButton = true,
+	excludeMainElementClassName,
```

Comment:
I think the naming of the prop could be simplified:
```suggestion
	excludeElementClassName,
```